webrtc.html

```diff
@@ -10932,9 +10932,11 @@ <h4>MediaTrackSupportedConstraints, MediaTrackCapabilities,
         <code><a>RTCSessionDescription</a></code> applied via
         <code>setRemoteDescription</code> and the actual RTP data. This means
         that certain members, such as <code>facingMode</code>,
-        <code>echoCancellation</code>, <code>latency</code>,
-        <code>deviceId</code> and <code>groupId</code>, will always be
-        missing.</p>
+        <code>echoCancellation</code>, <code>deviceId</code> and <code>groupId
+        </code>, will always be missing.</p>
+        <p>The <code>latency</code> setting on the <code>MediaStreamTrack</code>
+        sourced by an <code>RTCPeerConnection</code> describes the delay between
+        receiving a media packet and rendering that media.
```

Comment:
"describes the delay introduced by the RTCPeerConnection implementation (such as a jitter buffer delay)."

Clarify that this is for both audio and video (in mediacapture-main this is audio-only)